pkg/sql/schema_change_plan_node.go

```diff
@@ -43,7 +43,7 @@ import (
 func (p *planner) FormatAstAsRedactableString(
 	statement tree.Statement, annotations *tree.Annotations,
 ) redact.RedactableString {
-	return formatStmtKeyAsRedactableString(statement, annotations, tree.FmtSimple|tree.FmtOmitNameRedaction)
+	return formatStmtKeyAsRedactableString(statement, annotations, tree.FmtSimple)
```

Comment:
I don't quite understand how `FormatAstAsRedactableString` and `formatStmtKeyAsRedactableString` work together. We're making this call to `formatStmt` with `FmtSimple` option but then we set the `FmtCtx` to combine with the `FmtOmitNameRedaction`. What am I missing?

Reply:
`FormatAstAsRedactableString` is used within the schema changer, and the result is used to show the statement in EXPLAIN plans and other debugging interfaces.

`formatStmtKeyAsRedactableString` is used more broadly, including by SQL stmt logging code.

The change here is just to apply the `FmtOmitNameRedaction` within `formatStmtKeyAsRedactableString` instead of in the calling function.

`FmtOmitNameRedaction` was only added here in `FormatAstAsRedactableString` in the first commit of this PR.

or is your question about how combining the flags works? they are bitfields, so they can be OR'd together to have multiple flags apply.